bl31/plat_params.c

```diff
@@ -17,6 +18,13 @@ void bl31_plat_params_setup(void *from_bl2, bl31_plat_param_handler_t handler) {
 			continue;
 
 		switch (param->tag) {
+		case BL31_PLAT_PARAM_COREBOOT_TABLE:
+			if (COREBOOT)
```

Comment:
Yeah, I can change that if you like. Is that required ARM TF style?

I generally prefer this style over preprocessor #if for constants, because it reads nicer and it allows the compiler to syntax check even the code branch that is not enabled for the current build configuration (making it easier to prevent bit rot for rarely-used paths). But I can comply with your style if you want.